docs/manual/src/tutorial/Rust_scaffolding.md

```diff
@@ -0,0 +1,39 @@
+# Rust scaffolding
+
+## Rust scaffolding code
+
+Now we generate some Rust helper code to make the `add` method available to foreign-language bindings.  
+
+First, add `uniffi` to your crate dependencies: it is the runtime support code that powers uniffi's serialization of data types across languages:
+
+<!-- TODO: Use a published version -->
+```toml
+[dependencies]
+uniffi = { git = "https://github.com/mozilla/uniffi-rs", branch = "main" }
+```
+
+Then let's add `uniffi_build` to your build dependencies: it generates the Rust scaffolding code that exposes our Rust functions as a C abi.
```

Comment:
I recall thom having very specific opinions on how to describe this C layer, and I'm not sure "C abi" is technically accurate. Something like "C-compatible FFI layer" or similar might work.